doc/src/manual/embedding.md

```diff
@@ -1,22 +1,24 @@
 # Embedding Julia
 
-As we have seen in [Calling C and Fortran Code](@ref), Julia has a simple and efficient way to
-call functions written in C. But there are situations where the opposite is needed: calling Julia
-function from C code. This can be used to integrate Julia code into a larger C/C++ project, without
-the need to rewrite everything in C/C++. Julia has a C API to make this possible. As almost all
-programming languages have some way to call C functions, the Julia C API can also be used to build
-further language bridges (e.g. calling Julia from Python or C#).
+As we have seen in [Calling C and Fortran Code](@ref), Julia has a simple and efficient way
+to call functions written in C. But there are situations where the opposite is needed:
+calling Julia functions from C code. This can be used to integrate Julia code into a larger
+C/C++ project, without the need to rewrite everything in C/C++. Julia has a C API to make
+this possible. As almost all programming languages have some way to call C functions, the
+Julia C API can also be used to build further language bridges (e.g. calling Julia from
+Python or C#).
 
 ## High-Level Embedding
 
-__Note__: This section covers embedding Julia code in C on Unix-like operating systems. For doing
-this on Windows, please see the section following this.
+__Note__: This section covers embedding Julia code in C on Unix-like operating systems. For
+doing this on Windows, please see the section following this,
+[High-Level Embedding on Windows with Visual Studio](@ref).
 
 We start with a simple C program that initializes Julia and calls some Julia code:
 
 ```c
 #include <julia.h>
-JULIA_DEFINE_FAST_TLS() // only define this once, in an executable (not in a shared library) if you want fast code.
```

Comment:
Perhpas just keep this one as is, since the code base uses `//` for comments in the C based source files.